examples/libp2p-in-the-browser/README.md

```diff
@@ -1,22 +1,55 @@
-# libp2p running in the Browser
+# libp2p in the browser
 
-One of the primary goals with libp2p P2P was to get it fully working in the browser and interopable with the versions running in Go and in Node.js.
+This example leverages the [Parcel.js bundler](https://parceljs.org/) to compile and serve the libp2p code in the browser. Parcel uses [Babel](https://babeljs.io/) to handle transpilation of the code. You can leverage other bundlers such as Webpack or Browserify for development, but we will not be covering them here.
 
-# 1. Setting up a simple app that lists connections to other nodes
+## Setup
 
-Start by installing libp2p's dependencies.
+In order to run the example, first install the dependencies from same directory as this README:
```

Comment:
You also need to ensure dependencies in the root of the project are installed since it depends on `libp2p: "../../"`

Reply:
NPM should actually handle the install for you, I've verified this works a few times after clearing node modules from the project root.

Reply:
Oh nice, I did not realise.